metadata_tools/hosts/GO_0xxx/GO_0xxx_geometry_cloud.py

```
@@ -0,0 +1,74 @@
+import asyncio
+import os, sys
+from cloud_tasks.worker import Worker, WorkerData
+
+import metadata_tools.util as util
+import metadata_tools.common as com
+import host_config as hconf
+import geometry_config as config
+from metadata_tools.geometry_support import process_tables, get_args
+
+#========================================================================================
+def process_task(task_id: str,
```

Ruff (0.14.10)

Unused function argument: `task_id`

(ARG001)